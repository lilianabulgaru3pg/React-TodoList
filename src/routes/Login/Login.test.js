import React from 'react';
import { shallow } from 'enzyme';
import LoginPage from './index';

describe('LoginPage', () => {
  const wrapper = shallow(<LoginPage />);

  it('should render a <LoginForm />', () => {
    expect(wrapper.find('LoginForm')).toHaveLength(1);
  });

  it('LoginPage should render correctly', () =>
    expect(wrapper).toMatchSnapshot());
});
