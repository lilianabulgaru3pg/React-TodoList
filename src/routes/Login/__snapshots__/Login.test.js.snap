// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginPage LoginPage should render correctly 1`] = `
<Fragment>
  <main
    className="flex-item-1 center-content"
  >
    <LoginForm />
  </main>
  <aside
    className="flex-item-2 center-content right-card-1 blue-bg"
  >
    <h3
      className="card-label"
    >
      To-do List
    </h3>
  </aside>
</Fragment>
`;
